.github/workflows/cmake.yml

```
@@ -891,7 +891,10 @@ jobs:
 
     - name: Upload job coverage report to coveralls
       uses: coverallsapp/github-action@v2
-      if: (matrix.codecov && !contains(matrix.os, 'z15') && (env.COVERALLS_REPO_TOKEN != '' || github.repository == 'zlib-ng/zlib-ng'))
+      if: |
+        matrix.codecov
+        && !contains(matrix.os, 'z15')
```

actionlint (1.7.11)

[error] parser did not reach end of input after parsing the expression. 1 remaining token(s) in the input: ")"

(expression)